cubedash/themes/odc/templates/logo.html

```diff
@@ -0,0 +1 @@
+<img src="{{ theme_static('odc-logo-horizontal.svg') }}" height="26px" alt=""/>
```

Comment:
I might fiddle with this later. The logo looks out of place in the current colour scheme. Perhaps we'll switch to the monochrome/white version of the logo in the short term until colour theming is available.